src/frontend/tsx/Chessboard.tsx

```diff
@@ -374,6 +384,21 @@ export function Chessboard () {
     )
   }
 
+  const renderSpecialDrawPopup = (game: ActiveGameInfo) => {
+    return (
+      <Popup open={game.drawClaimAvailable}>
+        <div>
+          <p>{`You're eligible to claim a draw (threefold-repetition or 50 move rule)`}</p>
+          <br/>
+          <div className='draw-resolution row'>
+            <button className="accept" role="button" onClick={acceptSpecialDrawOnClick}>accept</button>
+            <button className="reject" role="button" onClick={declineSpecialDrawOnClick}>decline</button>
+          </div>
+        </div>
+      </Popup>
+    )
+  }
+
```

Comment:
I don't think this should be a pop-up. I think it should be a button in the game panel that's deactivated unless a special draw is available: if I remember correctly, that's how lichess and chess.com handle it when auto-accept-draw is disabled.